docs/document/docs/guides/tooltip-and-field-group-customization.md

```diff
@@ -0,0 +1,269 @@
+        {
+          label: 'Monday',
+          value: '1'
+        },
+        {
+          label: 'Tuesday',
+          value: '2'
+        },
+        {
+          label: 'Wednesday',
+          value: '3'
+        },
+        {
+          label: 'Thursday',
+          value: '4'
+        },
+        {
+          label: 'Friday',
+          value: '5'
+        }
+      ],
+      value: cellData
+    });
+  };
+
+  const renderLocation = cellData => {
+    return new Kuc.Dropdown({
+      items: [
+        {
+          label: '-----',
+          value: '-'
+        },
+        {
+          label: 'Tokyo',
+          value: 'Tokyo'
+        },
+        {
+          label: 'Vietnam',
+          value: 'Vietnam'
+        },
+        {
+          label: 'Shanghai',
+          value: 'Shanghai'
+        }
+      ],
+      value: cellData
+    });
+  };
+
+  const renderNote = cellData => {
+    return new Kuc.Text({
+      value: cellData
+    });
+  };
+
+  const renderAction = cellData => {
+      const button = document.createElement('button');
+      const tooltipForButton = new Kuc.Tooltip({
+        title: 'Submit changes or additions for this entry.',
+        container: button,
+        describeChild: false,
+        placement: 'bottom'
+      });
+      return tooltipForButton;
+    };
+
+  const table = new Kuc.Table({
+    label: 'team info',
+    columns: [
+      {
+        title: 'Name',
+        field: 'name',
+        render: renderName
+      },
+      {
+        title: 'In office day',
+        field: 'day',
+        render: renderDay
+      },
+      {
+        title: 'Location',
+        field: 'location',
+        render: renderLocation
+      },
+      {
+        title: 'Note',
+        field: 'note',
+        render: renderNote
+      },
+      {
+        title: 'Action',
+        field: 'action',
+        render: renderAction}
+    ],
+    data: [
+      {
+        name: '',
+        day: [],
+        location: '-',
+        note: ''
+      }
+    ]
+  });
+```
+
+#### Create FieldGroup with Table component
+Create a FieldGroup component with the label "KUC team" and the Table component as its content.
+
+```javascript
+  const fieldGroup = new Kuc.FieldGroup({
+    label: 'KUC team',
+    content: table
+  });
+```
+
+#### Create Tooltip with the FieldGroup
+Create Tooltip for the FieldGroup and associated them.
+
+```javascript
+  const tooltipForFieldGroup = new Kuc.Tooltip({
+    title: 'Office day info',
+    container: fieldGroup,
+  });
+```
+
+#### Add Event listener for FieldGroup 
+Add event listener for FieldGroup to change the tooltip message when FieldGroup is expanded or collapsed
+
+```javascript
+   fieldGroup.addEventListener('change', e => {
+    if (e.detail.expanded) {
+      tooltipForFieldGroup.title = '';
+    } else {
+      tooltipForFieldGroup.title = 'Office day info';
+    }
+  });
+```
+
+#### The display on UI
+![render](/img/tooltip_fieldgroup_customize.gif)
+
+
+### Using Two Types of Tooltip
```

Comment:
Please add notes that there is detailed explanation of describeChild here.
https://kintone-ui-component.netlify.app/docs/components/desktop/tooltip#property

Reply:
And you explained describeChild true and false pattern, but the important point we wanna mention is the purpose of using each.
- Set describeChild to false if the Tooltip represents the purpose of the element itself (provides a label for the element it is attached to).
- Set describeChild to true if the Tooltip represents the description of the element (provides additional information or a supplementary description about the element it is attached to).

Please update the example and explanation below so that user can understand it.